tests/foreman/cli/test_errata.py

```diff
@@ -464,15 +452,15 @@ def test_negative_install_by_hc_name_without_org_info(self):
 
         :CaseLevel: System
         """
-        with self.assertRaises(CLIReturnCodeError) as context:
+        with pytest.raises(CLIReturnCodeError) as context:
```

Comment:
https://docs.pytest.org/en/stable/reference.html#pytest-raises

`pytest.raises` takes a kwarg of `match`, which I think you should take advantage of here.  It looks like CLIReturnCodeError will include stderr in its `__str__` / `__repr__`, and `match` should work for your use case.

Then you don't have to separately assert outside of the pytest.raises context.

Reply:
Still ToDO

Reply:
Changes made